scripts/build/build_macos_arm64.sh

```diff
@@ -89,7 +89,7 @@ log "building wheels"
 # Build the binary packages
 export CIBW_PLATFORM=macos
 export CIBW_ARCHS=arm64
-export CIBW_BUILD='cp{38,39,310,311}-*'
+export CIBW_BUILD='cp{38,39,310,311,312}-*'
```

Comment:
So this is the part I had forgotten, sorry about that.

I managed to get a builder. Packages on the way.